### ISSUE TITLE
Anyone else having issues with HTTP error 403?

### ISSUE THREAD

**Issue by GreatOldOne** [charliewolf/pynder#175]
Only like and dislike are giving me those, they aren't working at all, but everything else is.
I tried debugging the Tinder client on a new installation with a new Facebook account and with a different IP address and even the official client is returning 403 for likes and dislikes.

Anyone else experiencing the same?

**Comment by GreatOldOne** [comment]
From more testing it seems to be some sort of block applied to new Facebook accounts.
Not sure however whether it's caused by lack of friends or by the date the Facebook profile was created.

**Comment by brenly** [comment]
I would be interested to see if this changes with a Facebook account that is older than a day, older than a week. Keep us posted! Thank you.

Has everything been working fine with your primary Facebook account?

**Comment by GreatOldOne** [comment]
@brenly Yeah, but my main Facebook account is 2 years old and has 50 friends.
My secondary Facebook account is now a day old and still getting 403 on like/dislike, superlike works though.
However, it seems I'm the only person getting this. No one here using throwaway Facebook accounts for Tinder?

**Comment by kkai4u** [comment]
@GreatOldOne  I am getting 403 as well, I'm able to login as well and the account I'm using is also a throwaway account but it is pretty nicely aged.
Also, I'm able to superlike as well but like throws a 403.

**Comment by GreatOldOne** [comment]
@kkai4u How old is the account? Does it have any friends? Is the email from a throwaway service or from a large company such as Gmail?

**Comment by kkai4u** [comment]
@GreatOldOne 
It's using a throwaway mail, I'm able to log into my facebook.
It's about a month old at least in my case.

**Comment by GreatOldOne** [comment]
Does it have any friends?

**Comment by kkai4u** [comment]
75 friends and 300+ pending requests.

**Comment by GreatOldOne** [comment]
Hmmm... I wonder if it's related to your email. If you don't mind, could you tell me the email domain for this account?

**Comment by kkai4u** [comment]
I logged into the account and tried swiping a few people, yours and my account are probably already flagged. I swiped a person who had superliked me and it did not show the person in my matches.

**Comment by GreatOldOne** [comment]
The block is as soon as you create the account, so it's automatic.
Since you've friends and it's a month old, it leaves us with the email address.
Would need to know the domain you are using for your throwaway to compare against mine (my main account which works fine is from Gmail).

**Comment by kkai4u** [comment]
The email address is a garbage one, ktlhqge.xyz

**Comment by GreatOldOne** [comment]
That's probably it, my throwaways were from disposable emails.
I'm a bit busy right now, could you try creating a Facebook account with a newly created Gmail account and see if you still get 403?

**Comment by GreatOldOne** [comment]
Yup, tried with a throwaway Gmail account and it worked.
*Sigh* Tinder... Why not ruin your product even further, now denying people with custom emails to use your app?

**Comment by kkai4u** [comment]
@GreatOldOne  I end up with a 
ValueError: No JSON object could be decoded
for some ids, is it because of my proxies or some other reason?

**Comment by GreatOldOne** [comment]
Never had it, I need to know what data you are receiving from the server for me to be able to figure it out.
In theory proxies could cause it by either corrupting the data or by sending something else to you instead.

**Comment by GustavPS** [comment]
Hi, Im having this problem as well but I am using a real phone number with a real sim card when I make the facebook account..

**Comment by RokasStepon** [comment]
Anyone else is having this issue even with fb accounts registered with gmail emails?

**Comment by rostyslav-matushkin** [comment]
This case is actual for me. Who can help?
